index.html

```diff
@@ -6,14 +6,94 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" type="text/css" href="css/grid.css">
+    <link rel="stylesheet" type="text/css" href="css/main.css">
     <!-- Add a link to your custom CSS here -->
 
     <title>Responsive grid project</title>
 </head>
 
 <body>
-
+        <div class="navigation-wrapper  col-12"> 
+                <div class="">
+                <div class="logotext col-6 sm-col-12">ALAN JOHNSON</div>
+                <div class="sublogo col-6 sm-col-12">Personal Life Coach</div>
+                </div>
+               
+                    <ul class="col=6" >
+                        <li>Home</li>
+                        <li>About</li>
+                        <li>Services</li>
+                        <li>Motivational Reading</li>
+                        <li>Blog</li>
+                        <li>Contact</li>
+                        <li>Log in</li>
+                    </ul>
+                </nav>
+            </div>
+    <div class="container2 col-12">
+    <img class="cover col-12" src="css/b74b2725ad015f5e_800x800ar.jpg"></img>
+   <div class="col-6 stamp">
+    <h2 class="margin0">Ambition is the first way towards</h2>
+    <h3 class="">SUCCESS</h3>
+    <button class="margin0">BOOK NOW</button>
+    </div>
+</div>
     <!-- Add your markup here -->
+    <div class="faceblock col-12">
+    <div class=" bio-wrap">
+        <div class="introtext col-6"> His blog post includes his very best tips—almost too good to give away for free. He’s also showcased other coaches, included quotes and drawn cartoons. Some might even call the post “epic.”
+
+            But doubt has crept up.
+            
+            “What if nobody wants to read my post, because my opening lines are crap?”
+            
+            Henry has read opening paragraphs that truly captivated him, and made him eager to read on. But now he has to be honest with himself … his own opening is not that good.
+        </div>
+        <img class="facepic col-6" src="css\james-stewart-man-person-actor-53487.jpeg">
```

Comment:
Don't use `col-...` classes on `img`s. Set `col-6` on a div, and move the image inside the div. The image should be set to `width: 100%`